i18n/ja.toml

```diff
@@ -0,0 +1,72 @@
+[read_more]
+other = "続きを読む"
+
+[copyright]
+other = "Copyright"
+
+[all_rights]
+other = "All Rights Reserved"
```

Comment:
Translation for this?

Reply:
In Japan, `Copyright` and `All Rights Reserved` will not translated (same as Chinese). So, I kept this.

You can check this with Japanese Government's homepage like [this](https://www.kunaicho.go.jp/) or [this](https://www.mlit.go.jp/).